$NetBSD: patch-configure,v 1.1 2014/11/01 15:59:35 nros Exp $
pkgsrc doesn't have lib64 so don't set it (fixes build on linux).
--- configure.orig	2014-11-01 14:04:18.108550253 +0000
+++ configure
@@ -15426,7 +15426,6 @@ if test "$enable_64bit" = "yes"; then
       *linux*)	if test "$GXX" = "yes"; then
 		   CXXFLAGS="$CXXFLAGS -m64"
 		fi
-		LIBSUFFIX=64
 
 		;;
       *hpux*)	if test "$host_cpu" = "ia64"; then
